recipes/qemu/meta.yaml

```
@@ -0,0 +1,99 @@
+{% set name = "qemu" %}
+{% set version = "5.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # if you download the source from GH archive it will be incomplete and you'll get
+  # ERROR: missing file $SRC_DIR/ui/keycodemapdb/README
+  #
+  # This is not a GIT checkout but module content appears to
+  # be missing. Do not use 'git archive' or GitHub download links
+  # to acquire QEMU source archives. Non-GIT builds are only
+  # supported with source archives linked from:
+  # 
+  #   https://www.qemu.org/download/#source
+  # 
+  # Developers working with GIT can use scripts/archive-source.sh
+  # if they need to create valid source archives.
+  #
+  #url: https://github.com/qemu/qemu/archive/refs/tags/v{{ version }}.tar.gz
+  url: https://download.qemu.org/qemu-{{ version }}.tar.xz
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

Add c++ compiler because there is 'optional' code that is being skipped.  Not sure what it is.